### PR TITLE
Completed Avatar

### PR DESCRIPTION
**Pull request by jsuryakt**
Please review avatar component
image

### CODE REVIEW

**Review by Balakumar3018 — APPROVED** [review]
Looks good to me. Avoid spaces between lines

File: minimal.css [hunk 1]
```diff
@@ -1,131 +1,69 @@
...
-    font-size: 3rem;
-}
-.icon-md {
-    font-size: 3rem;
-}
-.icon-lg {
-    font-size: 4rem;
+.avatar-lg {
+    width: 6.25rem;
+    /* 100px */
```
Comment (Koushith) [inline minimal.css:67]: you can remove these comments, if its not required,

**Review by Koushith — APPROVED** [review]
LGTM. good to merge

File: minimal.css (hunk 1 above)
Comment (jsuryakt) [reply minimal.css:67]: Sure

**Pull request merged**